conpot/conpot.cfg

```diff
@@ -39,6 +39,14 @@ host = taxiitest.mitre.org
 port = 80
 inbox_path = /services/inbox/default/
 use_https = False
+
+[stix]
+port_path_list = [
+    "//conpot_template/protocols/modbus/@port",
```

Comment:
No, we dont need them, will remove them from config.cfg and instead will only pass the protocols. I will fix this. :)